build/cmake/package_tester/modules/testing.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+if [ -z "${testing_sh_included}" ]
+then
+
+    testing_sh_included=1
+
+    desired_state() {
+        case $1 in
+            CLEAN )
+                :
+                ;;
+            INSTALLED )
+                install
+                ;;
+        esac
+    }
+
+    tests_healthy() {
+        cd /
+        fdbcli --exec status
+        if [ $? -ne 0 ]
+        then
+            return 1
+        fi
+        healthy="$(fdbcli --exec status | grep HEALTHY | wc -l)"
+        if [ -z "${healthy}" ]
+        then
+            __res=1
```

Comment:
This is inconsistent with the structure of the other usages of the `__res=1` idiom. Also there's a bug here: we need to return __res

Reply:
This is actually a good one. The standard idiom should really be used here as this function `cd`s into another directory. I fixed this.